packages/lab/src/stepped-tracker/TrackerStep/TrackerStep.tsx

```diff
@@ -19,7 +21,7 @@ import trackerStepCss from "./TrackerStep.css";
 
 const withBaseName = makePrefixer("saltTrackerStep");
 
-type State = "default" | "completed";
+type State = "default" | "completed" | "warning" | "error";
```

Comment:
Started a chat, around the naming of this prop

Reply:
Following dev-chat discussions, have switched to a more "presentational" style for the step component. Prop name is now `icon` (with `state` still supported as a deprecated fallback for now)